problem_builder/mentoring.py

```diff
@@ -948,6 +923,10 @@ def score(self):
         return Score(score, int(round(score * 100)), correct, incorrect, partially_correct)
 
     @property
+    def complete(self):
+        return not self.score.incorrect and not self.score.partially_correct
```

Comment:
@e-kolpakov Since this is nit-level: There is an upcoming task for expanding test coverage for Step Builder. That will be a good opportunity to address this. I'll add a note to the description of that task to make sure we won't forget.